### PR TITLE
Font

### PR DESCRIPTION
**Pull request by marcustisater**
Added FontFaceObserver with web fonts

### CODE REVIEW

**Comment by marcustisater** [comment]
:sleepy: any idea why this is failing @mxstbr ?

**Comment by marcustisater** [comment]
I forgot to use @font-face instead of link... will fix asap.

**Comment by marcustisater** [comment]
I need some help with this one, it says the document is not defined from FontFaceObserver app/index.js:15

I guess ESLint wants to assume that this is a selector for the browser?

**Comment by okonet** [comment]
I guess the `FontFaceObserver` requires `document` to be defined. But it's not in node when running tests.

**Comment by marcustisater** [comment]
Yeah that's what I meant to say. How can we go around this?

**Comment by marcustisater** [comment]
@MoOx may I ask for some assistance with this one? :blush:

File: web_modules/App/index.js [hunk 1]
```diff
@@ -8,6 +9,21 @@ import Footer from "Footer"
...
+}, () => {
+  document.body.classList.remove("merriweather-loaded")
+})
+
+FiraSansObserver.check().then(() => {
+  document.body.classList.add("fira-sans-loaded")
+}, () => {
+  document.body.classList.remove("fira-sans-loaded")
+})
+
```
Comment (MoOx) [inline web_modules/App/index.js]: Maybe try wrapping this code in an `if (typeof window !== "undefined") {`. If you still have an issue, try replacing the import of fontfaceobserver by a require in this new if window block.

**Comment by thangngoc89** [comment]
Wrap it in 

```js
if (typeof window === 'underfined')
```

**Comment by marcustisater** [comment]
Ah, that's one way to do it. Thanks guys :smile:

**Comment by MoOx** [comment]
That's the only real way for universal javascript from what I know.

**Comment by marcustisater** [comment]
Yeah, you are probably right. I'm not a javascript  expert :smile: 

I looked into if it was possible to ignore that particular code with ESLint browser:true but it runs through node when running tests.

**Comment by marcustisater** [comment]
I give up! :raised_hands: :triumph:

File: web_modules/App/index.js [hunk 2]
```diff
@@ -8,6 +8,25 @@ import Footer from "Footer"
 
 import styles from "./index.css"
 
+if (typeof window !== "undefined") {
+  const FontFaceObserver = require("FontFaceObserver")
```
Comment (MoOx) [inline web_modules/App/index.js]: typo, the module is lowercase ``fontfaceobserver``.

Comment (marcustisater) [inline web_modules/App/index.js]: :blush: Thank you.

**Pull request merged**